sycl/include/sycl/ext/oneapi/experimental/group_helpers_sorters.hpp

```diff
@@ -343,6 +343,41 @@ class group_sorter {
   }
 };
 
+template <typename T, typename U, typename CompareT = std::less<>,
+          std::size_t ElementsPerWorkItem = 1>
+class group_key_value_sorter {
+  CompareT comp;
+  sycl::span<std::byte> scratch;
+
+public:
+  template <std::size_t Extent>
+  group_key_value_sorter(sycl::span<std::byte, Extent> scratch_,
+                         CompareT comp_ = {})
+      : comp(comp_), scratch(scratch_) {}
+
+  template <typename Group>
+  std::tuple<T, U> operator()(Group g, T key, U value) {
+    static_assert(ElementsPerWorkItem == 1,
+                  "ElementsPerWorkItem must be equal 1");
+
+    using KeyValue = std::tuple<T, U>;
+    auto this_comp = this->comp;
+    auto comp_key_value = [this_comp](const KeyValue &lhs,
+                                      const KeyValue &rhs) {
+      return this_comp(std::get<0>(lhs), std::get<0>(rhs));
+    };
+    return group_sorter<KeyValue, decltype(comp_key_value),
+                        ElementsPerWorkItem>(scratch, comp_key_value)(
+        g, KeyValue(key, value));
+  }
+
+  static constexpr std::size_t memory_required(sycl::memory_scope scope,
```

Comment:
According to the spec `memory_required` is not `constexpr` for `default_sorters`. Please, make corresponding changes for all default sorters

Reply:
Thanks, fixed.